SWAGGER/views/pages/home.ejs

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>API Documentation AT14</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+  <style>
+    #content
+    {
+        margin: 0; padding: 0; height: 91%; overflow: hidden;                
+        position:absolute; left: 0; right: 0; 
+    }      
+  </style>
+</head>
+<body>
+
+<nav class="navbar navbar-expand-md bg-dark navbar-dark">
+  <a class="navbar-brand" href="#">
+    <img src="img/perro.png" alt="Logo" style="width:20px;">
+    API Doc
+  </a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavbar">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="collapsibleNavbar">
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <a class="nav-link" href="javascript:void(0);" onClick="urlChange('http://localhost:8000/MachineLearning')">Machine Learning</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="javascript:void(0);" onClick="urlChange('http://localhost:8000/VideoConverter')">Convert Service</a>
```

Comment:
Same here, it would be better if you can use a variable from the .env to call this URL